task4/requirements.txt

```diff
@@ -0,0 +1,3 @@
+redis
+flask
+requests
```

Comment:
better to use pip freeze with version details instead of just the name of packages

Reply:
OMG, I didn't notice that, and I have modified requirements file. Thanks for the heads-up.